src/routes/graphql/schemas.ts

```diff
@@ -18,3 +20,347 @@ export const createGqlResponseSchema = {
+            resolve: async (parent, args, { prisma }) => {
+                const posts = await prisma.post.findMany();
+                return posts;
+            },
+        },
+        post: {
+            type: PostType,
+            args: {
+                id: { type: UUIDType },
+            },
+            resolve: async (parent, args, { prisma }) => {
+                const { id } = args;
+                const post = await prisma.post.findUnique({ where: { id: id }});
+                return post || null;
+            },
+        },
+        profiles: {
+            type: new GraphQLList(ProfileType),
+            resolve: async (parent, args, { prisma }) => {
+                const profiles = await prisma.profile.findMany();
+                return profiles;
+            },
+        },
+        profile: {
+            type: ProfileType,
+            args: {
+                id: { type: UUIDType },
+            },
+            resolve: async (parent, args, { prisma }) => {
+                const { id } = args;
+                const profile = await prisma.profile.findUnique({ where: { id: id }});
+                return profile || null;
+            },
+        },
+        getProfileByUserId: {
+            type: ProfileType,
+            args: {
+                userId: { type: UUIDType },
+            },
+            resolve: async (parent, args, { prisma }) => {
+                const { userId } = args;
+                const profile = prisma.profile.findUnique({ where: { userId: userId }});
+                return profile || null;
+            },
+        },
+        memberTypes: {
+            type: new GraphQLList(MemberType),
+            resolve: async (parent, args, { prisma }) => {
+                const memberTypes = await prisma.memberType.findMany();
+                return memberTypes;
+            },
+        },
+        memberType: {
+            type: MemberType,
+            args: {
+                id: { type: MemberTypeId },
+            },
+            resolve: async (parent, args, { prisma }) => {
+                const { id } = args;
+                const memberType = await prisma.memberType.findUnique({ where: { id: id }});
+                return memberType || null;
+            },
+        },
+        /*prismaStats: {
+            type: PrismaStatsType,
+            resolve: async (fastify) => {
+                const prismaStats = await fastify.prismaStats
+                return { operationHistory: prismaStats } || null;
+            },
+        }, */
+        getPostByUserId: {
+            type: new GraphQLList(PostType),
+            args: {
+                userId: { type: UUIDType },
+            },
+            resolve: async (parent, args, { prisma }) => {
+                const { userId } = args;
+                const post = await prisma.post.findMany({ where: { authorId: userId }});
+                return post || null;
+            },
+        },
+        userSubscribedTo: {
```

Comment:
Please take a look at `src/routes/users/_userId/user-subscribed-to/index.ts` how prisma can be called in this call.